packages/core/test/integration/error.test.ts

```diff
@@ -711,7 +721,7 @@ describe(getTestDialectTeaser('Sequelize Errors'), () => {
       } catch (error) {
         expect(error).to.be.instanceOf(ForeignKeyConstraintError);
         assert(error instanceof ForeignKeyConstraintError);
-        if (dialect === 'sqlite3') {
+        if (dialect === 'sqlite3' || dialect === 'oracle') {
```

Comment:
Looking at `error.cause.message` we do get the index back. Why aren't we defining it?

Reply:
Fixed retrieving the constraint name.